app/views/transactionsAddReqResModal.html

```diff
@@ -1,127 +1,180 @@
+<style>
```

Comment:
Ideally yes. Problem with putting this into the stylesheet is that it will apply for ALL modals. By doing it the way i did it with these modals it injects those styles and overrides the defaults to make the modals full height/width. This is only need on the transaction modals.

I can look at finding a better way for this